docs/sql/dml.txt

```diff
@@ -143,6 +143,67 @@ Resulting partitions of the last insert by query::
    ``limit``, ``offset`` and ``order by`` are not supported inside the
    query statement.
 
+On Duplicate Key Update
+-----------------------
+
+The ``ON DUPLICATE KEY UPDATE`` clause is used to update the existing
+row if inserting is not possible because of a duplicate-key conflict::
+
```

Comment:
maybe add a select upfront, so users see, what rows are in the table, so they see, that a duplicate key conflict will occur